rails-snow-queen/app/views/quotes/new.html.erb

```diff
@@ -1,17 +1,25 @@
-<h1>Quote calculator</h1>
+<div class="d-flex row">
+   <div id="panel" class="col-4 p-5">
+      <h1>Quote calculator</h1>
+      <p>Get a free quote for a salting or snow clearing service</p>
 
-<p>Get a free quote for a salting or snow clearing service</p>
+      <form action="" id="AddressSearch">
+         <label for="address">Search your location:</label><br>
+         <input id="address" type="textbox" placeholder="123 Street Rd, Port Moody" size="40">
+         <input id="submit" type="submit" value="Find" class="btn btn-light btn-sm">
```

Comment:
Might want to ["Bootstrapify"](https://getbootstrap.com/docs/4.1/components/forms/) the form to make it look nice

Reply:
Prettification of the app hasn't been implemented yet, just did some light improvements to avoid visual migraines